packages/matter-node.js/src/matter/interaction/SubscriptionHandler.ts

```diff
@@ -5,18 +5,20 @@
  */
 
 import { MatterDevice } from "../MatterDevice";
-import { InteractionServerMessenger } from "./InteractionMessenger";
+import { InteractionServerMessenger, StatusResponseError } from "./InteractionMessenger";
 import { Fabric } from "../fabric/Fabric";
 import { AttributeWithPath, AttributePath, INTERACTION_PROTOCOL_ID, attributePathToId } from "./InteractionServer";
 import { Time, Timer } from "../../time/Time";
 import { NodeId } from "../common/NodeId";
 import { TlvSchema } from "@project-chip/matter.js";
+import { tryCatchAsync } from "../../error/TryCatchHandler";
+import { StatusCode } from "./InteractionMessages";
 import { Logger } from "../../log/Logger";
 import { SecureSession } from "../session/SecureSession";
 
 const logger = Logger.get("SubscriptionHandler");
 
-const SUBSCRIPTION_MAX_INTERVAL_PUBLISHER_LIMIT_MS = 1000 * 60 * 60; /** 1 hour */
+const SUBSCRIPTION_MAX_INTERVAL_PUBLISHER_LIMIT_MS = 3 * 60 * 1000; //1000 * 60 * 60; /** 1 hour */
```

Comment:
Why did this change?

Reply:
I will add a comment before I merge. 

In fact former behavior is as per spec. But this leads to the effect that if you restart the device the controllers wait the agreed max amount of time to resubscribe (because for them all is fine). This can be a long time and I found that not making sense in real usage because the device could so stay "unchanged" for up to 45 mins in our implementation.

That's why I lowered to 3 mins. 

Additionally inward experiencing that Chiptool is also not using the 1h max - Chiptool uses the "max time" send by device as max, so updates could come really often if the device requests so. This is also not making too much sense because so you spam the network with empty subscriptions info messages. That's why I tried to do something in the middle.